packages/bootstrap-4/test/addButton.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
Can you capitalize all the file names of tests (since they're components too, for consistency)?